internal/server/http/test/testdata/imposters/test_request.imp.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Please, extend this example with all the valid attributes in order to ensure all of them works properly with YAML.